components/camel-rest-swagger/src/main/java/org/apache/camel/component/rest/swagger/RestSwaggerEndpoint.java

```diff
@@ -651,4 +705,18 @@ static String queryParameterExpression(final Parameter parameter) {
         return expression.toString();
     }
 
+    protected static HttpRequestBase createMethod(String url) {
```

Comment:
I'd rather save up on this function and write it as one line
```suggestion
    protected static HttpRequestBase createMethod(String url) {
        return new HttpGetWithBodyMethod(url, new StringEntity("", ""));
    }
```

Reply:
I agree,  I haven't refactored this part of code (I was removing unnecessary code and at this moment method is completely useless)